### ISSUE TITLE
Add more loss functions & ability to change the decoder output

### ISSUE THREAD

**Issue by alitinet** [theislab/multimil#24]
Add new loss functions for each data type:
1. For the RNA-seq modality (see https://github.com/theislab/scarchesp/blob/master/scarchest/models/trvae/losses.py):
    - [x] negative binomial (NB)
    - [x] zero-inflated negative binomial. (ZINB)
2. for ATAC-seq:
    - [x] binary cross entropy (BCE)

Also, for NB, ZINB and BCE, we need unnormalized counts, hence:
- [x] allow the user to specify the layer of the input adatas.
